### PR TITLE
QA-81: GitLab Pipeline completition

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
@mzedel Could you briefly look at this one? Note my decision on building/publishing only the docker version.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -15,22 +13,43 @@ stages:
...
+    - build
   script:
-    - echo "publish mender-stress-test-client binary to s3 here"
\ No newline at end of file
+    - docker load -i image.tar
+    - docker tag $SERVICE_IMAGE $DOCKER_REPOSITORY:$COMMIT_TAG
+    - docker tag $SERVICE_IMAGE $DOCKER_REPOSITORY:$CI_COMMIT_REF_SLUG
+    - echo -n $DOCKER_HUB_PASSWORD | docker login -u $DOCKER_HUB_USERNAME --password-stdin
+    - docker push $DOCKER_REPOSITORY:$COMMIT_TAG
+    - docker push $SERVICE_IMAGE
```
Comment (mzedel) [inline .gitlab-ci.yml]: aren't you pushing both here? IMO that doesn't hurt, but considering that this is more of a devtool only the `DOCKER_TAG` might be enough... you can get to the `COMMIT_TAG` anyway through the `DOCKER_TAG` + image hash 🤷‍♂

**Comment by mzedel** [comment]
LGTM as is - the additional tag won't hurt us, just _maybe_ docker hub 😁

File: .gitlab-ci.yml (hunk 1 above)
Comment (lluiscampos) [reply .gitlab-ci.yml]: I used the same pattern in all the repositories (most of them already merged!).

**Pull request merged**